cfgmgr/macsecmgr.cpp

```diff
@@ -699,7 +767,7 @@ bool MACsecMgr::configureMACsec(
             port_name,
             network_id,
             "mka_cak",
-            profile.primary_cak);
+            decodeKey(profile.primary_cak, profile.cipher_suite));
```

Comment:
can we make this change backward compatible ?

Reply:
sure Abhishek - I have created an issue : https://github.com/sonic-net/sonic-swss/issues/2899 to follow up the requirement to have a config_db option to specifiy the mode.